README.md

```diff
@@ -1,31 +1,27 @@
-![logo](https://cloud.githubusercontent.com/assets/3964980/7551665/c133ce00-f65f-11e4-8d65-f4f122880b1c.png)
+![logo](https://cloud.githubusercontent.com/assets/3964980/11156885/6fa1967a-8a1c-11e5-8c78-e552ffba31c0.png)
 
 [![Join the chat at https://gitter.im/WPIRoboticsProjects/GRIP](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/WPIRoboticsProjects/GRIP?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 [![Build Status](https://travis-ci.org/WPIRoboticsProjects/GRIP.svg?branch=master)](https://travis-ci.org/WPIRoboticsProjects/GRIP)
+[![Build status](https://ci.appveyor.com/api/projects/status/9xl8pggec4l75pqb/branch/master?svg=true)](https://ci.appveyor.com/project/ThomasJClark/grip/branch/master)
 [![codecov.io](http://codecov.io/github/WPIRoboticsProjects/GRIP/coverage.svg?branch=master)](http://codecov.io/github/WPIRoboticsProjects/GRIP?branch=master)
 
+
 # GRIP Computer Vision Engine
 
 GRIP (the Graphically Represented Image Processing engine) is a program for rapidly prototyping and deploying computer
-vision algorithms, primarily for robotics applications.  **It is currently in a very early state of development**,
+vision algorithms, primarily for robotics applications.  **It is currently in an early state of development**,
 should be ready in time for the 2016 FIRST Robotics Competition.
 
-## ALPHA TESTERS
+## Getting Started
 
 1. Download the newest [release](https://github.com/WPIRoboticsProjects/GRIP/releases) for your operating system.
 2. Run the included installer.
 3. Open GRIP
 
-## DEVELOPERS
-
-To run, use the included Gradle wrapper script.
+Check out the release notes and [the wiki](https://github.com/WPIRoboticsProjects/GRIP/wiki) for more information.
 
-    ./gradlew run
+## Building GRIP From Source
 
-## Docs
-Javadocs for the project can be found [here](http://WPIRoboticsProjects.github.io/GRIP)
+To build and run, use the included Gradle wrapper script.
 
-## Common Problems
-### UnsatisfiedLinkError
-The opencv library for linux requires version 4.8 or higher of gcc in order for `GOMP_4.0` to be present.
-Check the version of gcc you have installed using `gcc -v` and update if necessary.
```

Comment:
If you are going to remove this then please add a DEVELOPER.md and put this there.